frontend/containers/discover-map/map-help/component.tsx

```diff
@@ -0,0 +1,207 @@
+          <h3 className="font-serif text-3xl font-semibold text-green-dark">
+            <FormattedMessage defaultMessage="Help" id="SENRqu" />
+          </h3>
+        </div>
+        <div className="px-6 mt-32 mb-24 sm:px-10 sm:mb-40">
+          <p>
+            <FormattedMessage
+              defaultMessage="The map allows you to quickly see the location of each project and understand the context of the different projects.<n>Projects</n> Each project is represented by a marker that has a color defined by the project category."
+              id="WocAcK"
+              values={{
+                n: (chunk: string) => <span className="block my-2 font-semibold">{chunk}</span>,
+              }}
+            />
+          </p>
+          <div className="flex flex-wrap items-center justify-between gap-4 mt-7">
+            <div className="flex items-center gap-x-4">
+              {Object.values(theme.colors.category).map((categoryColor) => (
+                <MapPin
+                  className="stroke-transparent"
+                  style={{ fill: categoryColor }}
+                  key={categoryColor}
+                  aria-label={formatMessage({ defaultMessage: 'Project pin icon', id: 'AMKznU' })}
+                />
+              ))}
+              <p className="text-sm">
+                <FormattedMessage defaultMessage="Project marker" id="wvp0Qm" />
+              </p>
+            </div>
+            <div className="flex items-center gap-x-4">
+              <Icon
+                aria-label={formatMessage({ defaultMessage: 'Project cluster icon', id: 'XwwnKr' })}
+                icon={clusterExemple}
+                className="h-14 w-14 drop-shadow"
+              />
+              <p className="text-sm">
+                <FormattedMessage defaultMessage="Group of projects" id="wghPR3" />
+              </p>
+            </div>
+          </div>
+
+          <div className="mt-12">
+            <div className="flex items-center justify-between mb-5">
+              <p className="font-semibold">
+                <FormattedMessage defaultMessage="Map Layers" id="A8VNfu" />
+              </p>
+              <div className="px-2 py-1 rounded shadow right-10">
+                <Icon
+                  aria-label={formatMessage({ defaultMessage: 'Map layers icon', id: 'A4FcXS' })}
+                  icon={Layers}
+                  className="inline-block w-4 h-4 mr-2.5"
+                />
+                <span className="text-sm text-gray-800">
+                  <FormattedMessage defaultMessage="Map Layers" id="A8VNfu" />
+                </span>
+              </div>
+            </div>
+            <p>
+              <FormattedMessage
+                defaultMessage="You can activate a varity of data layers to help you to understand the context of each project. These layers may be of 3 types:"
+                id="FPr3FX"
+              />
+            </p>
+            <ul className="!list-disc list-inside mt-6 space-y-1">
```

Comment:
This makes it slightly closer to the design:
```suggestion
            <ul className="pl-5 mt-6 space-y-1 list-disc">
```